project/graphql/translator_testing_model.graphql

```diff
@@ -87,34 +96,17 @@ type OneHopTestSuite
     testSuiteSpecification: TestSuiteSpecification
   }
 
-interface PathfinderPathNode
+type PathfinderPathNode
   {
     ids: [String]
     name: String
   }
 
 type PathfinderTestAsset
   {
+    id: Uriorcurie!
     name: String
     description: String
-    inputId: Uriorcurie
-    inputName: String
-    inputCategory: ConceptCategory
-    predicateId: PredicateType
-    predicateName: String
```

Comment:
I think the predicate related fields are actually necessary and included within the test assets.